Trappist/src/Promact.Trappist.Repository/Categories/CategoryRepository.cs

```diff
@@ -26,33 +26,45 @@ public CategoryRepository(TrappistDbContext dbContext)
         }
 
         /// <summary>
-        /// Adding a Category in Category model
+        /// Method to Add a Category
         /// </summary>
-        /// <param name="catagory">Object of class Category</param>
-        public void AddCategory(Category category)
+        /// <param name="catagory">Task</param>
+        public async Task AddCategoryAsync(Category category)
         {
-            _dbContext.Category.Add(category);
-            _dbContext.SaveChanges();
+            await _dbContext.Category.AddAsync(category);
+            await _dbContext.SaveChangesAsync();
         }
 
         /// <summary>
-        /// Find a Respective Id from Catagory Table
+        /// Method to Get Category by its Id
```

Comment:
the method is not written in a proper case. Please write proper English statements.